dessia_common/forms.py

```diff
@@ -308,25 +308,24 @@ def generate_from_text(cls, stream: StringFile):
 
     def add_standalone_object(self, object_: StandaloneBuiltinsSubobject):
         """
-        Add a standalone object to object_list.
-
-        It doesn't return anything, hence, API will update object when computing from frontend.
+        Adds a standalone object to object_list.
```

Comment:
We discussed with Tanguy, and it looks like convention sais the verb should not have an "s" at the end